src/test/utils/macUtils copy.ts

```diff
@@ -0,0 +1,58 @@
+'use strict';
```

Comment:
is this file a slightly modified copy of src/test/utils/macUtils.ts? Do we need both files?

Reply:
No I can remove this

Reply:
Addressed in the latest commit, Pls check